File: Problem_sets/sentimental-mario-more/mario.py
from cs50 import get_int

# Prompts user for a height
while True:
    height = get_int("Height: ")
    if height >= 1 and height <= 8:
        break

# Prints spaces to align the left pyramid and the first hash
for i in range(height):
    for n in range(height - 1 - i):
        print(" ", end="")
    print("#", end="")

    # Prints the rest of hashes for the first pyramid + double space
    for m in range(i):
        print("#", end="")
    print("  ", end="")

    # Prints the right pyramid
    for k in range(i + 1):
        print("#", end="")
    print()

File: Problem_sets/mario-more/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int n;
    do
    {
        n = get_int("Enter the height: ");
    }
    while
    (n < 1 || n > 8);

    for (int i = 0; i < n; i++)
    {
        for (int k = 0; k < (n - 1 - i); k++)   // prints space each time it's necessary to Right-align the triangle correctly
        {
            printf(" ");
        }
        printf("#");                            // prints "#" in a loop until var i == height input by user


        for (int j = 0; j < i; j++)     // writes an additional "#" on the same row with each NEXT cycle until equal to var i
        {
            printf("#");
        }

        printf("  ");                   // prints out space between pyramids

        for (int l = 0; l <= i; l++)    // compare amount of "#" written in the left part and print if less or same amount
        {
            printf("#");
        }
        printf("\n");                   // after finishing 1 cycle we move onto the next row
    }



}